docs-kits/kits/Manufacturing as a Service Kit/page_adoption-view.md

```diff
@@ -0,0 +1,101 @@
+---
+id: Adoption View MaaS Kit
+title: Adoption View
+description: 'MaaS Kit'
+sidebar_position: 2
+---
+
+## Vision & Mission
+
+**Enable interoperable, federated network of networks linking manufacturing demand with manufacturing capabilities for digitally enabling manufacturing as a service.**
+
+**Comprehensive reference architecture with standardized interfaces for on-demand manufacturing**
+
+The Manufacturing-as-a-Service (MaaS) KIT defines the mechanisms and services required for an interoperable, federated network of networks where manufacturing demand meets manufacturing capabilities. The Catena-X Digital Manufacturing as a Service Ecosystem (CDME) provides a comprehensive reference architecture with standardised interfaces and data models, including first implemented services, to present the operational network of networks with a single entry point into the world of domain-independent on-demand manufacturing (ODM). The main focus is on the mediation of manufacturing demand and manufacturing supply, which is offered in a service- orientated manner. In addition, IT services enable connectivity and process automation to be part of this digital marketplace.
+
+## Business Value
+
+### Reference architecture
+
+The reference architecture includes standardized and tested interfaces and data models that can be used for the development and implementation of Manufacturing-as-a-Service solutions (Chapter 4 of Whitepaper).
+
+### Standards
+
+Standardizing the necessary interfaces and data models of our architecture enables to exchange data with different systems and users and to extend the reference architecture with additional services, replace certain individual services with equivalent ones or to remove optional services.
+
+Within the Resiliency KiT Manufacturing as a service, two main standards were defined:
+
+- Manufacturing Capability Model and API - for declaration of offered and required manufacturing capabilities
+- Request for Quotation Model and API - for manufacturing demand configuration
```

Comment:
Can you link to the standards or if they are not published - please provide the number, the name and the relevant version and link to the standard library. 

In addition the standards should be h2 and one of the last points on the adoption view.

Reply:
Section has been moved and links/numbers have been provided